src/pypromice/process/utilities.py

```diff
@@ -41,7 +41,15 @@ def reformat_lon(dataset, exempt=['UWN', 'Roof_GEUS', 'Roof_PROMICE']):
     '''Switch gps_lon to negative values (degrees_east). We do this here, and 
     NOT in addMeta, otherwise we switch back to positive when calling getMeta 
     in joinL2'''
-    if dataset.attrs['station_id'] not in exempt:
+    if 'station_id' in dataset.attrs.keys():
+        id = dataset.attrs['station_id']
+    else:
+        id = dataset.attrs['site_id']
+
+    if id not in exempt:
+        if 'gps_lon' not in dataset.keys():
+            print("?????????", id, "missing gps_lon")
```

Comment:
Is this a warning or debugging message?
Please user logging and write a more explicit messages.

Reply:
It was for debugging. Will be removed